### PR TITLE
 Rename all classes

### PR DESCRIPTION
**Pull request by truongthanh2606**
Remove prefix "Anakin"
Add prefix "Addressbook" to AddressBook's classes
Change directory hierarchy
Change SortCommand.java

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -47,9 +47,9 @@
     protected Ui ui;
     protected Config config;
     protected UserPrefs userPrefs;
-    protected AnakinLogic logic;
-    protected AnakinModel model;
-    protected Storage storage;
+    protected Logic logic;
+    protected Model model;
+    protected AnakinStorage anakinStorage;
```
Comment (tomforge) [inline src/main/java/seedu/address/MainApp.java]: Kind of weird that the main app components are `Logic`, `Model`, `UI`, but for storage it's `AnakinStorage`.. i don't think you should've swapped the `AnakinStorage` and `Storage` interfaces

File: src/main/java/seedu/address/MainApp.java [hunk 2]
```diff
@@ -18,22 +18,22 @@
...
+import seedu.address.storage.JsonUserPrefsStorage;
 import seedu.address.storage.Storage;
 import seedu.address.storage.StorageManager;
-import seedu.address.storage.XmlAnakinStorage;
-import seedu.address.storage.JsonUserPrefsStorage;
 import seedu.address.storage.UserPrefsStorage;
+import seedu.address.storage.XmlStorage;
 import seedu.address.ui.Ui;
-import seedu.address.ui.UiManager;
+import seedu.address.ui.UiManagerAddressbook;
```
Comment (tomforge) [inline src/main/java/seedu/address/MainApp.java]: `UiManager` looks like it already implements anakin's `UI`, why rename it to `UiManagerAddressBook`?

File: src/main/java/seedu/address/commons/core/AddressbookComponentManager.java [hunk 3]
```diff
@@ -7,17 +7,17 @@
  *
  * Registers the class' event handlers in eventsCenter
  */
-public class AnakinComponentManager {
+public abstract class AddressbookComponentManager {
```
Comment (tomforge) [inline src/main/java/seedu/address/commons/core/AddressbookComponentManager.java]: `AnakinComponentManager` was a duplicate of `ComponentManager`. We should delete it and also change `AnakinModelManager` (which is also the only place it's used) to extend `ComponentManager` instead, like how `LogicManager`, `UIManager`, and `StorageManager` all do

File: src/main/java/seedu/address/commons/core/ComponentManager.java [hunk 4]
```diff
@@ -7,7 +7,7 @@
  *
  * Registers the class' event handlers in eventsCenter
  */
-public abstract class ComponentManager {
+public class ComponentManager {
```
Comment (tomforge) [inline src/main/java/seedu/address/commons/core/ComponentManager.java]: See comment on `AnakinComponentManager`

File: src/main/java/seedu/address/commons/core/AddressbookMessages.java [hunk 5]
```diff
@@ -3,12 +3,16 @@
 /**
  * Container for user visible messages.
  */
-public class AnakinMessages {
+public class AddressbookMessages {
```
Comment (tomforge) [inline src/main/java/seedu/address/commons/core/AddressbookMessages.java]: We should merge `AnakinMessages` and `Messages` and just use `Messages` from now on

File: src/main/java/seedu/address/logic/AddressbookLogic.java [hunk 6]
```diff
@@ -4,13 +4,12 @@
...
-import seedu.address.model.anakindeck.AnakinCard;
-import seedu.address.model.anakindeck.AnakinDeck;
+import seedu.address.model.person.Person;
 
 /**
- * API of the Logic component
+ * API of the AddressbookLogic component
  */
-public interface AnakinLogic {
+public interface AddressbookLogic {
```
Comment (tomforge) [inline src/main/java/seedu/address/logic/AddressbookLogic.java]: Looks like the anakin specific functions have been ported over to `Logic` already. We should be able to delete this file?

File: src/main/java/seedu/address/commons/core/AddressbookComponentManager.java (hunk 3 above)
Comment (truongthanh2606) [reply src/main/java/seedu/address/commons/core/AddressbookComponentManager.java]: I didn't rename this class directly, might be a side-effect of renaming another class. Resolved

File: src/main/java/seedu/address/logic/AddressbookLogic.java (hunk 6 above)
Comment (truongthanh2606) [reply src/main/java/seedu/address/logic/AddressbookLogic.java]: I leave the delete part until we finish integration

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 7]
```diff
@@ -5,14 +5,10 @@
...
 
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
-    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
-
-    // Anakin messages
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_DECK_DISPLAYED_INDEX = "The deck index provided is invalid";
+    public static final String MESSAGE_INVALID_DECK_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
```
Comment (Lessthanfree) [inline src/main/java/seedu/address/commons/core/Messages.java]: Why Persons Listed?

File: src/main/java/seedu/address/logic/Logic.java [hunk 8]
```diff
@@ -4,10 +4,11 @@
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.Person;
+import seedu.address.model.anakindeck.Card;
+import seedu.address.model.anakindeck.Deck;
 
 /**
- * API of the Logic component
+ * API of the AddressbookLogic component
```
Comment (Lessthanfree) [inline src/main/java/seedu/address/logic/Logic.java]: AddressbookLogic?

**Pull request merged**